README.md

```diff
@@ -0,0 +1,42 @@
+# cross_modality_query
+
+Server-side code for HuBMAP Cells API.
+[Python](https://github.com/hubmapconsortium/hubmap-api-py-client) and 
+[Javascript](https://github.com/hubmapconsortium/hubmap-api-js-client) clients available.
+
+## Usage
+
+This is only a high-level outline of the API:
+See the [Python client examples](https://github.com/hubmapconsortium/hubmap-api-py-client/tree/main/examples) for details.
+
+The API supports a small number of basic operations which can be combined to construct more complex queries.
+The API runs at a **`base_url`**: Currently `https://cells.dev.hubmapconsortium.org/api` is available.
+
+Issuing a `POST` to `{base_url}/{output_type}/` (where `output_type` is `cell`, `organ`, `gene`, or `cluster`),
+with query parameters in the body, will return a **query set handle**.
```

Comment:
Is there any harm in calling these "query"/"query handle" rather than "query set"/"query set" handle? Is the use of "query set" a direct mapping from the Django ORM?

Reply:
I personally don't have a strong feeling about this.  The use of query set was actually more of a name collision with the Django ORM object.

Reply:
I will update to "query"/"query handle".